File: API/ContaCentral/ContaCentral/Domain/Models/Carteira.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace ContaCentral.Domain.Models
{
    public class Carteira
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public decimal Saldo { get; set; }
        public Boolean Ativo { get; set; }
        public Boolean Principal { get; set; }
        public Guid UserId { get; set; }
        public ApplicationUser User { get; set; }

        [JsonIgnore]
        public List<Transacao> ListaTransacao { get; set; }

    }
}

File: Website/src/app/domain/models/carteira.ts
export class Carteira{

	id : number = 0
	nome: string = ""
	saldo: number = 0
	userId: string = ""
	ativo: boolean = null
	principal:boolean = null

	constructor(id:number, nome: string, saldo: number, userId: string, ativo: boolean, principal: boolean) {
		//specify your own constructor logic
		this.id = id;
		this.nome = nome;
		this.saldo = saldo;
		this.userId = userId;
		this.ativo = ativo;
		this.principal = principal;
	}
	
}